packages/cypress/src/task/index.ts

```diff
@@ -69,9 +70,16 @@ const loadAllCustomEntityConfigs = async (
             isCustomViewConfig ? 'View' : 'Application'
           } config for ${packageInfo.packageJson.name}`
         );
+
+        const customEntityConfigCacheKey = isCustomViewConfig
+          ? `${processedConfig.env.entryPointUriPath}-${path.basename(
+              packageInfo.dir
+            )}`
+          : processedConfig.env.entryPointUriPath;
```

Comment:
During monorepo traversal by `loadAllCustomEntityConfigs`, any discovered custom view config is cached using a key derived from the directory name (equivalent to the template name)